config/default/manager_image_patch.yaml

```diff
@@ -8,5 +8,5 @@ spec:
     spec:
       containers:
       # Change the value of image field below to your controller image URL
-      - image: gcr.io/k8s-staging-cluster-api/capk-manager:dev
+      - image: 127.0.0.1:49158/capk-manager-amd64:dev
```

Comment:
I'm not what effects this would have
@agradouski @cchengleo is this fine by you to change it permanently?

Reply:
Done